.github/workflows/generate-data.yaml

```
@@ -191,7 +191,7 @@ jobs:
           ls -larth
           mkdir -p out/
           prefix="https://raw.githubusercontent.com/${{ needs.check.outputs.repo }}/refs/heads"
-          oracle="${prefix}/oracle-data/oracle-output.csv"
+          oracle="${prefix}/target-data/oracle-output.csv"
```

I'm actually not sure how this worked in the first place 😵 

```suggestion
          oracle="${prefix}/main/target-data/oracle-output.csv"
```